injector/network_disruption.go

```diff
@@ -68,7 +68,7 @@ func NewNetworkDisruptionInjector(spec v1beta1.NetworkDisruptionSpec, config Net
 }
 
 // Inject injects the given network disruption into the given container
-func (i networkDisruptionInjector) Inject() error {
+func (i networkDisruptionInjector) Inject(ctx context.Context) error {
```

Comment:
Wanted to seek your advice on where in `applyOperations` to check for cancellation and end early